### PR TITLE
66 question removal

### PR DESCRIPTION
**Pull request by JeToKo**
## Describe your changes

Describe here

## Checklist before requesting a review

-   [X] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added thorough tests

### CODE REVIEW

File: frontend/src/components/PollCreationPage.tsx [hunk 1]
```diff
@@ -151,6 +155,15 @@ function PollCreationPage() {
                 {' '}
                 Submit Poll{' '}
             </Button>
+            <Button
+                variant="contained"
+                onClick={cancelHandler}
+                className="cancel-poll-btn"
+                sx={{ mt: '1rem', width: 200 }}
```
Comment (Marakke) [inline frontend/src/components/PollCreationPage.tsx:162]: I would like these inline stylings to be in SCSS file

**Comment by Marakke** [comment]
Otherwise looks and works great 👍

**Review by vrindavj — DISMISSED** [review]
Approved

**Review by Marakke — DISMISSED** [review]

**Review by Marakke — APPROVED** [review]

**Review by Marakke — DISMISSED** [review]

**Review by vrindavj — DISMISSED** [review]
Approved

**Review by oonasaksman — DISMISSED** [review]

**Review by oonasaksman — APPROVED** [review]

**Review by Marakke — APPROVED** [review]

**Pull request merged**